### PR TITLE
Reopen of #9162

### PR DESCRIPTION
**Pull request by taukausanake**
Redid the swarmers eating active clone pods things that was stale. Partly because I think it makes sense but MAINLY because I wanted to use the web editor. :^) Fite me

🆑alexkar598 and taukausanake
tweak: Swarmers can no longer destroy active clone pods.
/🆑

### CODE REVIEW

File: code/game/gamemodes/miniantags/bot_swarm/swarmer.dm [hunk 1]
```diff
@@ -306,6 +306,11 @@
 /obj/spacepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	to_chat(S, "<span class='warning'>Destroying this vehicle would destroy us. Aborting.</span>")
 
+/obj/machinery/clonepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(occupant)
+		to_chat(S, "<span class='warning'>Destroying this machine while it is occupied would result in biological and sentient resources to be harmed. Aborting.</span>")
+		return
+
```
Comment (FlattestGuitar) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: `..()`

Comment (Crazylemon64) [reply code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: Oh, that's true
Thank you for this catch

File: code/game/gamemodes/miniantags/bot_swarm/swarmer.dm [hunk 2]
```diff
@@ -306,6 +306,11 @@
 /obj/spacepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	to_chat(S, "<span class='warning'>Destroying this vehicle would destroy us. Aborting.</span>")
 
+/obj/machinery/clonepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(occupant)
+		to_chat(S, "<span class='warning'>Destroying this machine while it is occupied would result in biological and sentient resources to be harmed. Aborting.</span>")
+		return ..()
```
Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: you don't want to `return ..()`

You want to call `..()` if there isn't a `return`

Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: you don't want to return ..()

You want to call ..() if there isn't a return

What you have means that the only way a swarmer can deconstruct a clonepod is if it does have an occupant in it.

Comment (taukausanake) [reply code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: Oh I get it. I'm dumb

**Pull request merged**